### PR TITLE
WD-9490 - update /download/server/thank-you

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Rebrand /download/server/thank-you-s390x - fix heading classes.
Last section will be redesigned separately

## QA

- [demo link](https://ubuntu-com-13692.demos.haus/download/server/thank-you-s390x)
- [figma](https://www.figma.com/file/cVGimTPHky4h3BPXfrT6GB/24.04-U.com---download-(rebranding)?node-id=522%3A29620&mode=dev)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-9490](https://warthogs.atlassian.net/browse/WD-9490)

[WD-9490]: https://warthogs.atlassian.net/browse/WD-9490?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
Now that I ca nsee the demo, these p-sections are not needed. But please implement a medium breakpoint layout like so:
image

File: templates/download/server/thank-you-s390x.html [hunk 1]
```diff
@@ -27,22 +27,22 @@ <h1 class="p-section--shallow">Thank you for downloading Ubuntu for LinuxONE and
...
-    <hr class="p-rule" />
     <div class="col-3">
       <h2 class="p-heading--2">Resources</h2>
     </div>
-    <div class="col-3">
-      <p class="p-heading--5 p-section--shallow">eBook</p>
-      <h3 class="p-heading--6"><a href="https://pages.ubuntu.com/Forrester_TAP_Report_2016.html">Forrester Research &mdash; get more from the cloud</a></h2>
+    <div class="col-3 col-medium-2">
+      <h3 class="p-heading--5">eBook</h3>
+      <p><a href="https://pages.ubuntu.com/Forrester_TAP_Report_2016.html">Forrester Research - get more from the cloud</a></p>
```
Comment (mtruj013) [inline templates/download/server/thank-you-s390x.html:37]:
```suggestion
      <p><a href="https://pages.ubuntu.com/Forrester_TAP_Report_2016.html">Forrester Research &mdash; get more from the cloud</a></p>
```

File: templates/download/server/thank-you-s390x.html [hunk 2]
```diff
@@ -27,22 +27,22 @@ <h1 class="p-section--shallow">Thank you for downloading Ubuntu for LinuxONE and
...
     </div>
-    <div class="col-3">
-      <p class="p-heading--5 p-section--shallow">Whitepaper</p>
-      <h3 class="p-heading--6"><a href="https://www.ibm.com/account/reg/us-en/signup?formid=urx-32554">IDC &mdash; next generation applications on Ubuntu for LinuxONE/z</a></h2>
+    <div class="col-3 col-medium-2">
+      <h3 class="p-heading--5">Whitepaper</h3>
+      <p><a href="http://www-01.ibm.com/marketing/iwm/dre/signup?source=stg-web&S_PKG=ov47989&S_TACT=C47305CW">IDC - next generation applications on Ubuntu for LinuxONE/z</a></p>
```
Comment (mtruj013) [inline templates/download/server/thank-you-s390x.html:41]:
```suggestion
      <p><a href="http://www-01.ibm.com/marketing/iwm/dre/signup?source=stg-web&S_PKG=ov47989&S_TACT=C47305CW">IDC &mdash; next generation applications on Ubuntu for LinuxONE/z</a></p>
```

**Review by mtruj013 — APPROVED** [review]
LGTM, just some minor changes

**Comment by carkod** [comment]
oh guys shouldn't this go to the download bubble?

**Pull request merged**